### PR TITLE
Update idx.session.expired error message

### PR DESCRIPTION
**Pull request by amirbeygi-okta**
Resolves: OKTA-423103

## Description:

Update idx.session.expired error message from `The session has expired.` to `You have been logged out due to inactivity. Refresh or return to the sign in screen.`

## PR Checklist

- [ ] Have you verified the basic functionality for this change?

- [ ] Added unit tests?

- [ ] Added e2e tests

- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?

### Reviewers:
@kamaldeepcheema-okta 

### Issue:

- [OKTA-423103](https://oktainc.atlassian.net/browse/OKTA-423103)

### CODE REVIEW

**Comment by kamaldeepcheema-okta** [comment]
we should also ensure that this string is not used in any of our selenium tests in the monolith

**Comment by amirbeygi-okta** [comment]
@kamaldeepcheema-okta What we would have to do if it was used there too ?
This is good enough to make sure it is not used in selenium ?
image

**Comment by amirbeygi-okta** [comment]
image

**Comment by kamaldeepcheema-okta** [comment]
> @kamaldeepcheema-okta What we would have to do if it was used there too ?
> This is good enough to make sure it is not used in selenium ?
> image

@amirbeygi-okta if we need to update the strings in okta-core, we would need to create a PR for okta-core with those updates. That won't be merged until the next version of the SIW with your changes here gets pulled in. You'll essentially be working with the SIW coordinator to ensure the monolith tests are updated appropriately at the time of SIW release.

The first step is to see if this change breaks any test in okta-core.

**Review by kamaldeepcheema-okta — APPROVED** [review]
Changes look good here, we just have to ensure that we update okta-core tests if needed

**Comment by pradeepdewda-okta** [comment]
Before merging we should run a downstream on bacon.. This may break selenium.

**Comment by amirbeygi-okta** [comment]
> Before merging we should run a downstream on bacon.. This may break selenium.
Looks good
https://okta.slack.com/archives/CCA77QVL2/p1630552895207300?thread_ts=1630430199.144000&cid=CCA77QVL2

**Comment by amirbeygi-okta** [comment]
OKTA-CORE PR
https://github.com/okta/okta-core/pull/56709

**Review by pradeepdewda-okta — APPROVED** [review]

File: packages/@okta/i18n/src/properties/login.properties [hunk 1]
```diff
@@ -1043,7 +1043,7 @@ oie.selfservice.unlock_user.failed.message = We are unable to unlock your accoun
 oie.selfservice.user.unlock.not.allowed = Self Service Unlock is not allowed at this time. Please contact support for assistance.
 
 ## Terminal errors
-idx.session.expired = The session has expired.
+idx.session.expired = You have been logged out due to inactivity. Refresh or return to the sign in screen.
```
Comment (jmelberg-okta) [inline packages/@okta/i18n/src/properties/login.properties:1046]: Can we have the IDX team's UX representative approve this messaging update in this PR please?

**Review by jmelberg-okta — APPROVED** [review]
This update LGTM! Also, please make sure to modify your commit message to adhere to our [contributing guidelines](https://github.com/okta/okta-signin-widget/blob/master/CONTRIBUTING.md#commit-message-guidelines) before merging into `master`.

**Review by nikhilvenkatraman-okta — APPROVED** [review]

File: packages/@okta/i18n/src/properties/login.properties (hunk 1 above)
Comment (amirbeygi-okta) [reply packages/@okta/i18n/src/properties/login.properties:1046]: We have confirmation from @jessicaallen-okta https://okta.slack.com/archives/CCA77QVL2/p1630440429159500?thread_ts=1630430199.144000&cid=CCA77QVL2

**Pull request closed without merging**